### ISSUE TITLE
Hosted Images Being Returned as Text/XML

### ISSUE THREAD

**Issue by ekarkosApollo** [jchristn/WatsonWebserver#24]
Watson Webserver version: 2.1.3
.NET Framework: 4.6.2

We are having an issue where images in a directory that we host up via `server.ContentRoutes.Add("/img/" + nameOfImage)` are being returned with `text/xml` headers when our other application accesses them.

**Comment by jchristn** [comment]
Hi @ekarkosApollo could you send me an example full URL that is being requested?  Thanks!

**Comment by ekarkosApollo** [comment]
So the way it works is we copy the image from its original location to a temporary cache folder next to the executable, the path to the image stored in the temporary cache would be something like `<path_to_exe>\img\foo.jpg` and they get hosted up on `localhost:9001/img/foo.jpg`.

Another separate application then grabs the image data from that URL for uploading to another web server, but for some reason that second application is getting the MIME type back as `text/xml`.

**Comment by jchristn** [comment]
Ok - interesting.  Within ```ContentRouteProcessor.cs``` it normalizes the file name/path and then calls ```GetContentType``` which uses the ```MimeTypes``` static class internally to retrieve the content-type based on the file extension.

In the case of ```foo.jpg``` it should be resolving to ```.jpg``` which should return ```image/jpeg``` for the content-type.  

You're certain that the full URL always ends in .jpg?

**Comment by ekarkosApollo** [comment]
It ends in the extension of the image. So for example, `foo.jpg` would be `localhost:9001/img/foo.jpg` but `bar.tif` would be `localhost:9001/img/bar.tif`.

**Comment by jchristn** [comment]
Got it. Do you notice this for all image types (extensions) or just certain ones?

**Comment by ekarkosApollo** [comment]
Yes we have noticed this for all file extensions.

The second application makes a `GET` request to my application hosting up the image. I ran a debugger to check the variable storing the incoming image blob, and this is what it came up with in the debugger:
![image](https://user-images.githubusercontent.com/40398579/62395536-c84a9e80-b52d-11e9-8da1-8a1cb98f7d76.png)

**Comment by jchristn** [comment]
Also do you see the ```Content-Type``` header reflected as ```text/xml``` when using ```curl -v```?  If you could provide some sample full URLs that would be helpful.  The only extensions that should be returning ```text/xml``` are:

```
        {".AddIn", "text/xml"}, 
        {".disco", "text/xml"}, 
        {".dll.config", "text/xml"}, 
        {".dtd", "text/xml"},
        {".dtsConfig", "text/xml"}, 
        {".exe.config", "text/xml"}, 
        {".mno", "text/xml"}, 
        {".SSISDeploymentManifest", "text/xml"}, 
        {".vml", "text/xml"}, 
        {".vsct", "text/xml"}, 
        {".vsixlangpack", "text/xml"},
        {".vsixmanifest", "text/xml"}, 
        {".vssettings", "text/xml"},  
        {".vstemplate", "text/xml"}, 
        {".wsdl", "text/xml"}, 
        {".xml", "text/xml"}, 
        {".xrm-ms", "text/xml"}, 
        {".xsd", "text/xml"},
        {".xsf", "text/xml"},
        {".xsl", "text/xml"},
        {".xslt", "text/xml"}, 
```

And anything not recognized should show up as ```application/octet-stream```

**Comment by jchristn** [comment]
Ok you definitely found a bug.  

I just tested using ```watson.jpg``` within a content route and it's not setting the content-type.

I will investigate this ASAP.  Thanks for letting me know!
![image](https://user-images.githubusercontent.com/1421081/62395756-0fcd2c80-b526-11e9-83d8-1355d650faaa.png)

**Comment by jchristn** [comment]
Just pushed a fix to NuGet in v2.1.4: https://www.nuget.org/packages/Watson/2.1.4

Commit is here: https://github.com/jchristn/WatsonWebserver/commit/38fbe8eccc5d7f17ea26abad9de25bd22315d6a4

Should only take ~10 minutes for NuGet to update, could you please let me know if this fixes the issue for you?

Thanks again for filing it!

**Comment by ekarkosApollo** [comment]
Looks like this fixed the issue. Thanks again for the quick responses.
